UPGRADING_v2.md

```
@@ -0,0 +1,62 @@
+	restServer,
+	grpcgatewayServer,
+)
+```
+
+## Upgrading from v0.50.x to v2
+
+Upgrading directly from v0.50.x to v2 is supported.
+Modules should be updated to support all the latest changes in the SDK.
+
+Read the module section from the v0.52 [UPGRADING.md](UPGRADING.md) file for more information.
+Then simply follow the instructions from the [v0.52 section](#upgrading-from-v052x-to-v200) from this file.
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)